app/frontend/styles/application.scss

```diff
@@ -36,7 +36,7 @@ $govuk-image-url-function: frontend-image-url;
 @import 'components/og-preview';
 @import 'components/pagination';
 @import 'components/school';
-@import 'components/school';
+@import 'components/schools_in_your_trust';
```

Comment:
Can we start to be more consistent with the naming here? I prefer `file-name.scss` compared to `file_name.scss` for frontend files